src/main/kotlin/study/racingcar/entity/Car.kt

```diff
@@ -4,11 +4,10 @@ class Car(private var moveCount: Int = 0) {
     val currentPosition: Int
         get() = moveCount
 
-    fun move(distance: Int) {
-        if (distance > 3) moveCount ++
+    fun move(
+        distance: Int,
+        validCondition: Int,
+    ) {
+        if (distance >= validCondition) moveCount++
```

Comment:
결국 이 행위도 두 값의 비교로 행동이 결정된다는 점에서는 확장성이 떨어집니다. 다음과 같이 구현할 수 없을까요?
```suggestion
    fun moves(strategy: MoveStrategy) {
        if (strategy.movable()) {
            moveCount++
        }
```